dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/src/main/app/"
+    schedule:
+      interval: "weekly"
```

Comment:
Good to add our team als reviewers to every 'updates' section so we can see these PRs automatically:

```    
reviewers:
      - "infonl/dimpact"
```

Reply:
Added and added some schedule stuff (monday morning so we have the whole week to review the prs)